multidimensional_urlencode/urlencoder.py

```diff
@@ -67,6 +70,6 @@ def urlencode(params):
         if isinstance(value, (list, tuple)):
             name += "[]"
 
-        url_params[name] = value
+        url_params.append((name, value))
```

Comment:
yes, for the tests to pass.
python2 returned consistently ordered list, but python3's `.item()` ordering is arbitrary.

[link 1](https://docs.python.org/3.0/whatsnew/3.0.html#views-and-iterators-instead-of-lists)
[link 2](http://stackoverflow.com/questions/14956313/dictionary-ordering-non-deterministic-in-python3)